src/Microsoft.AspNetCore.Authentication.Cookies/breakingchanges.netcore.json

```diff
@@ -0,0 +1,6 @@
+  [
+    {
+      "TypeId": "public class Microsoft.AspNetCore.Authentication.Cookies.CookieAuthenticationHandler : Microsoft.AspNetCore.Authentication.AuthenticationHandler<Microsoft.AspNetCore.Authentication.Cookies.CookieAuthenticationOptions>, Microsoft.AspNetCore.Authentication.IAuthenticationSignInHandler",
```

Comment:
Breaking change?

Reply:
New base class, shouldn't be a real breaking change.

Old signature derived from AuthenticationHandler
Now it derives from SignInAuthenticationHandler who continues to have a virtual SignOut method.

Reply:
Oh so not a real breaking change? Bug in the tool?

Reply:
The tool has always worked this way as far as I can tell, it just baselines the signature of all the public types/classes, so if you modify the inheritance or move stuff around it will generate diffs that require exceptions

Reply:
It probably is a bit smarter than that, but it does have limitations, so this one here is fine as long as it isn't a real breaking change.